src/Azure.Functions.Cli/StaticResources/funcignore

```diff
@@ -5,3 +5,8 @@
 local.settings.json
 test
 getting_started.md
+node_modules/.bin/
```

Comment:
I'm a bit nervous about excluding this one because literally any node module could put something in here, not just dev dependencies. I'd rather leave it off for now.